x-pack/plugins/security_solution/public/common/utils/download_blob.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Method for downloading any file
+ *
+ * @param blob raw data
+ * @param filename of file to be downloaded
+ *
+ */
+export const downloadBlob = (blob: Blob, filename: string) => {
+  const objectURL = window.URL.createObjectURL(blob);
+  const anchor = document.createElement('a');
```

Comment:
Surprised you don't have to append this to the body somewhere. In other use cases online I see people appending the anchor. This works across browsers, great, was just interesting to me.